src/main/java/com/treasuredata/underwrap/UnderwrapServer.java

```diff
@@ -75,6 +76,11 @@ public UnderwrapServer(Class<? extends UnderwrapApplication> applicationClass, P
         this.serverRootPath = serverRootPath;
     }
 
+    public void disableAccessLog()
```

Comment:
`setAccessLogEnabled` sounds a more common name to me.

Reply:
I will update so.